ledfx/api/virtual_effects.py

```diff
@@ -201,6 +210,16 @@ async def post(self, virtual_id, request) -> web.Response:
         effect_config = data.get("config")
         if effect_config is None:
             effect_config = {}
+            # if we already have this effect in effects then load it up
+            for virtual_cfg in self._ledfx.config["virtuals"]:
+                if virtual_cfg["id"] == virtual_id:
+                    if "effects" in virtual_cfg:
+                        if effect_type in virtual_cfg["effects"]:
+                            effect_config = virtual_cfg["effects"][
+                                effect_type
```

Comment:
This horror story of a layout came from PR workflow reformatting? Have I done something weird to trigger this getting mangled so bad...

Reply:
It's just black - you can turn it on/off by using 
# fmt: off
and then 
# fmt: on

We use it to enforce a consistent code style, but sometimes it can do some wonky things.

I'd just refactor it to condense the expression a bit and then tell it to ignore the block of code

Reply:
is it 88 as the default line length? Seems short, but will tweak, I think my pycharm default is 120

Reply:
PEP-8 dictates 80 characters.
Black uses 88 which they seem to think results in better files.
I like black just because it removes a lot of variability in the codebase and keeps things consistentish.

Reply:
yep, I refactored with a next iterator, and that pulled it under the 88